js/google_ima.js

```diff
@@ -52,6 +52,7 @@ require("../html5-common/js/utils/utils.js");
       //Constants
       var DEFAULT_IMA_IFRAME_Z_INDEX = 10004;
       var DEFAULT_ADS_REQUEST_TIME_OUT = 15000;
+      var DEFAULT_LOAD_VIDEO_TIME_OUT = 15000;
```

Comment:
What about the ad load timeout in the AMC.  IMA's load timeout and request timeout together are longer.  Won't these conflict?